tests/pom.xml

```diff
@@ -233,20 +206,11 @@
                         </goals>
                         <phase>test</phase>
                         <configuration>
-                            <systemProperties combine.children="append">
-                                <property>
-                                    <name>test.fpl</name>
-                                    <value>${test.fpl}</value>
-                                </property>
-                                <property>
-                                    <name>test.patch.product</name>
-                                    <value>${test.patch.product}</value>
-                                </property>
-                                <property>
-                                    <name>test.health</name>
-                                    <value>microprofile-health</value>
-                                </property>
-                            </systemProperties>
+                            <systemPropertyVariables>
```

Comment:
missing `combine.children="append"`

Reply:
I am wandering if that is really required, systemPropertyVariables seems to have this behavior by default.

Reply:
good point, I agree, @RoyalKarma please disregard my comments